ui/component/channelSelector/view.jsx

```diff
@@ -38,28 +38,26 @@ function ChannelSelector(props: Props) {
   }
 
   return (
-    <div>
```

Comment:
I removed this because I don't think it's necessary to have, if that's not the case please let me know :v: